### DISCUSSION TITLE
Adding notes to class repo

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#26] [Q&A] [answered]
Task 4 suggests adding notes to the class repo. I'm still unsure how to do this. Google was not helpful. Am I missing a notes section somewhere?

**Comment by bst3ck** [comment]
@Skye404 - I think we can do that in 2 different areas. We can add content to the Reference folder by adding new files and then submitting a pull request. We can also add comments to the discussion board like this one. Both are ways to add note to the class repo.

**Reply by Skye404** [reply]
I'm not seeing a References folder. I posted in the main discussion for now. Thanks.

**Reply by bst3ck** [reply, marked as answer]
It took me a minute to find it as well. When I click on Repos, I select the class and then on the main page it shows a list of file folders. You should be able to see the reference folder there.

![image](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/13371999/cf53e281-9cd3-43ea-9e0b-072c75e2974f)

Here is the direct link if that helps as well:
https://github.com/USU-ITLS-x265-23-summer/class-repo/tree/main/References

**Reply by Skye404** [reply]
Thanks! This is so helpful.
